commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java

```diff
@@ -61,6 +62,10 @@ public int doStartTag() {
 				putValue("orderId", commerceOrder.getCommerceOrderId());
 				putValue(
 					"commerceAccountId", commerceOrder.getCommerceAccountId());
+				putValue(
+					"workflowStatus",
+					WorkflowConstants.getStatusLabel(
```

Comment:
@gianmarcobrunialti  use the getStatus int not the label

Reply:
Done 👍